client/src/app/auth/forgot-password/page.tsx

```diff
@@ -0,0 +1,126 @@
+"use client";
+import { useState } from "react";
+import useFetch from "@/src/hooks/useFetch";
+import { AuthService } from "@/src/services/AuthService";
+import Link from "next/link";
+import Navbar from "@/src/components/Navbar/Navbar";
+import Footer from "@/src/components/Footer/Footer";
+import useThemeStore from "@/src/store/themeStore";
+import Image from "next/image";
+import ForgotPasswordSvg from "@/public/assets/Login.svg";
+
+const ForgotPassword = () => {
+  const theme = useThemeStore((state) => state.theme);
+  const [email, setEmail] = useState("");
+  const [errorMessage, setErrorMessage] = useState("");
+  const [successMessage, setSuccessMessage] = useState("");
+
+  const [{ isLoading }, forgotPasswordAPI] = useFetch(null);
+
+  const onHandleSubmit = () => {
+    if (email === "") {
+      setErrorMessage("Enter Email");
+    } else {
+      setErrorMessage("");
+      forgotPasswordAPI(() =>
+        AuthService.forgotPassword({ email }).then((response) => {
```

Comment:
We don't need to use then & catch, it will be handled by custom hook only that why we created to don't repeat it. Reason using custom hook is follow code/design principles. Also using custom hooks, you can get loading and error callbacks too. 

See the login or other pages to get reference. If you need any help, let me know.